SpEC/com_motion.py

```diff
@@ -208,10 +208,8 @@ def remove_avg_com_motion(path_to_waveform_h5='rhOverM_Asymptotic_GeometricUnits
     # Read the waveform data in
     w_m = read_from_h5(path_to_waveform_h5)
 
-    # Insert '_CoM' into the original `path_to_waveform_h5`; this will be the output path
-    path_to_new_waveform_h5 = path_to_waveform_h5.split('.h5', 1)
-    path_to_new_waveform_h5.insert(1, '_CoM.h5')
-    path_to_new_waveform_h5 = ''.join(path_to_new_waveform_h5).replace(w_m.descriptor_string + '_', '')
```

Comment:
If a piece of code cannot handle input data, I would hope such code will generate a clear error message (or just fail).  As such, using the output filename as the indicator of success seems unintuitive to me, especially, if there is no other warning nearby that this is happening.

What I found particularly confusing were implict assumptions of the code.  Specifically, the dependency of descriptor_string on the file-name.  (I wondered 'what if the descriptor_string is NOT contained in the filename?  Or what if somebody decides to modify the descriptor_string, e.g. making it more verbose?').

Anyway, replace(descriptor_string,...) added back.